packages/text-annotator/src/utils/cloneEvents.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Events need to be manually mapped into new objects:
+ * 1. It preserves the `target` and `currentTarget` properties.
+ *    Otherwise, they will be `null` when the event is read beyond the handler.
+ * 2. Spread operator can copy only own enumerable properties, not inherited ones.
+ *    Therefore, we need to manually copy the props we're interested in.
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget
+ * @see https://github.com/recogito/text-annotator-js/commit/65d13f3108c429311cf8c2523f6babbbc946013d#r144041390
+ */
+
+export const clonePointerEvent = (event: PointerEvent): PointerEvent => ({
+  ...event,
```

Comment:
Even though spreading `event` returns only the `{ isTrusted }` prop - it tricks the Typescript into thinking that the "cloned event" has the `PointerEvent` type